src/resmom_win/start_exec.c

```diff
@@ -3105,6 +3043,7 @@ start_process(task *ptask, char **argv, char **envp, bool nodemux)
 	if (pjob->ji_hJob == NULL) {
```

Comment:
Please use log_errf and log the jobid and name
https://github.com/PBSPro/pbspro/blob/06f24712f1b101a386d5b88656a24489250e52f0/src/resmom_win/start_exec.c#L3012